CRM/Upgrade/Incremental/sql/5.64.alpha1.mysql.tpl

```diff
@@ -15,3 +15,24 @@ ALTER TABLE civicrm_uf_group
 CHANGE `post_URL` `post_url` varchar(255) DEFAULT NULL COMMENT 'Redirect to URL on submit.',
 CHANGE `cancel_URL` `cancel_url` varchar(255) DEFAULT NULL COMMENT 'Redirect to URL when Cancel button clicked.'
 ;
+
+-- Ensure new name field is not null/unique. Setting to ID is a bit lazy - but it works.
+UPDATE civicrm_group SET `name` = `id` WHERE name IS NULL;
+
+-- Add name field, make frontend_title required (in conjunction with php function)
+{if $multilingual}
+    {foreach from=$locales item=locale}
+      UPDATE `civicrm_group`
+      SET `frontend_title_{$locale}` = `title_{$locale}`,
+      `frontend_description_{$locale}` = `description`
+      WHERE `frontend_description_{$locale}` IS NULL OR `frontend_description_{$locale}` = '';
```

Comment:
@eileenmcnaughton doesn't this need to be 2 separate queries to handle where frontend_description is NULL or an empty string

Reply:
@seamuslee001 I don't think so - or rather I'm not following why it would